starters/express-apollo-prisma/src/queue/job-generator-handler.spec.ts

```diff
@@ -0,0 +1,93 @@
+import amqplib from 'amqplib';
+import { createJobGeneratorHandler } from './job-generator-handler';
+
+const MOCK_URL = 'amqp://localhost:5673';
+
+const SPY_AMQPLIB_CONNECT = jest.spyOn(amqplib, 'connect');
+
+describe('.createJobGeneratorHandler', () => {
+	let req: any;
+	let res: any;
+	let mockCreateQueueChannel: any;
+	const next = jest.fn();
+	const ORIGINAL_ENV = process.env;
+
+	beforeAll(() => {
+		req = {
+			body: {
+				message: 'job message',
+			},
+		};
+		res = {
+			send: jest.fn(),
+			status: jest.fn().mockReturnThis(),
+		};
+		process.env = {
+			AMQP_URL: MOCK_URL,
+		};
+	});
+
+	afterAll(() => {
+		process.env = OLD_ENV;
```

Comment:
this throws error: should be ORIGINAL_ENV

Reply:
yeah i know

Reply:
i just commited the sugggestion